Tasks/UseRubyVersionV0/Tests/L0SetPathOnLinux.ts

```diff
@@ -1,41 +1,34 @@
-import ma = require('vsts-task-lib/mock-answer');
-import tmrm = require('vsts-task-lib/mock-run');
-import path = require('path');
-import fs = require('fs');
-import os = require('os');
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import * as tmrm from 'vsts-task-lib/mock-run';
 
 let taskPath = path.join(__dirname, '..', 'main.js');
 let tr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
 
 tr.setInput('versionSpec', '2.4');
 tr.setInput('addToPath', 'true');
 
-// provide answers for task mock
-let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
-    
-};
-tr.setAnswers(a);
-
 tr.registerMock('vsts-task-tool-lib/tool', {
     findLocalTool: () => path.join('/', 'Ruby', '2.4.4'),
     prependPath: (s: string) => {
         console.log('##vso[task.prependpath]' + s);
-   }
+    }
 });
 
-fs.symlinkSync = (s) => {
-}
-fs.unlinkSync = (s) => {
-}
-fs.existsSync = (s) => {
-    return true;
-}
-tr.registerMock('fs', fs);
+tr.registerMock('fs', {
+    symlinkSync: () => { },
+    unlinkSync: () => { },
+    existsSync: () => { return true; },
+    statSync: () => fs.statSync,
```

Comment:
I think statSync, writeFileSync, readFileSync must not be used.  The mocks here wouldn't work as expected if called -- they're functions that are returning a function object :)

Reply:
They need to be mocked, but should fall through.